docs/advanced-usage.md

```diff
@@ -22,7 +22,7 @@ steps:
 - uses: actions/checkout@v2
 - uses: actions/setup-java@v2
   with:
-    distribution: 'adopt'
+    distribution: 'adopt' # See 'Supported distributions' for available options @ README.md
```

Comment:
Why not change it to `adopt-hotspot` to [make it less ambiguous](https://github.com/actions/setup-java/issues/160)?

Reply:
I agree, that makes sense